Tests/NimbleTests/Matchers/ElementsEqualTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ElementsEqualTests.swift
+//  Nimble
+//
+//  Created by Luciano Almeida on 7/1/18.
+//  Copyright © 2018 Jeff Hui. All rights reserved.
+//
+
+import Foundation
+
+import Foundation
+import XCTest
+import Nimble
+
+final class ElementsEqualTest: XCTestCase, XCTestCaseProvider {
+
+    func testSequenceElementsEqual() {
```

Comment:
Prefer function name -> `testSequenceElementsEquality`, just like functions name in `EqualTest`.